.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 repos:
 - repo: https://github.com/Yelp/detect-secrets
-  rev: f6027a0521e044ba46e54611cabd787b7a88d1a9 
+  rev: v1.4.0
   hooks:
     - id: detect-secrets
-      args: ['--baseline', '.secrets.baseline']
-      exclude: package.lock.json
+      args: [ '--baseline', '.secrets.baseline' ]
+      exclude: package.lock.json
```

Comment:
This isn't a Node project so this exclusion isn't necessary (and is also the wrong file name!)